manifests/a/AmeyaP/UTAU/0.4.18/AmeyaP.UTAU.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: AmeyaP.UTAU
+PackageVersion: 0.4.18
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: utau0418inst.exe
+InstallerSwitches:
+  Custom: /norestart

```

Comment:
This is causing a command line error when trying to install the program. You will need to fix the installer switches so it installs unattended